nixos/tests/nix-store-gcs-proxy.nix

```diff
@@ -0,0 +1,20 @@
+import ./make-test-python.nix ({ pkgs, ... }:
+
+{
+  name = "nix-store-gcs-proxy";
+  meta = with pkgs.stdenv.lib.maintainers; {
+    maintainers = [ mrkkrp ];
+  };
+
+  machine = { pkgs, ... }: {
+    services.nix-store-gcs-proxy = {
+      something = {
+        address = "localhost:3000";
+      };
+    };
+  };
+
+  testScript = ''
+  start_all()
```

Comment:
this test isn't really helpful. It should probably at least try to see if the process

a) did not just die after starting
b) is listening on the configured port/address
c) delivers some kind of static health check response?!

Reply:
I'm afraid it's hard to make the test helpful unless there is a way to fake GCP credentials. The `nix-store-gcs-proxy` executable fails if valid credentials are not provided. cc @zimbatm